coremltools/test/api/test_api_examples.py

```diff
@@ -731,6 +731,131 @@ def test_unsanitized_input_name_during_prediction():
         error_str = str(error_info.value)
         assert "does not match any of the model input" in error_str
 
+    @staticmethod
+    @pytest.mark.skipif(not ct.utils._is_macos(), reason="Platform is not Mac OS")
+    def test_unsanitized_input_dtype_during_prediction():
```

Comment:
There is a lot of duplicated code across these five test methods. Let's just have one test method (call it `test_unsanitized_prediction_input`) that way we don't have to keep defining the program.

Reply:
@TobyRoseman 
Okay, I parameterize this test.